code/modules/mob/living/living.dm

```diff
@@ -511,7 +511,6 @@
 		GLOB.alive_mob_list += src
 		set_suicide(FALSE)
 		stat = UNCONSCIOUS //the mob starts unconscious,
-		blind_eyes(1)
```

Comment:
I see you removing these but not calling update_blindness anywhere, will it update properly on death ?

Reply:
It's no longer needed because death calls updatehealth()  which then calls update_stat() and that then called cure_blind(UNCONSCIOUS_BLIND) which cures the blind trauma if they happen to have it

Reply:
I don't see this working for living -> dead transition, since that early returns before blindness changes.

Reply:
I see it gets cured on revival but it's there on death, i'll add a line before the early return to remove the blindness trait